python/mxnet/gluon/rnn/rnn_cell.py

```diff
@@ -441,6 +441,9 @@ class LSTMCell(HybridRecurrentCell):
     params : Parameter or None
         Container for weight sharing between cells.
         Created if `None`.
+    in_transform_activation_type : str
```

Comment:
This name is too verbose. They are usually called activation and recurrent activation.
activation is applied to both input and next c